src/mixins/contact-mixin.ts

```diff
@@ -100,7 +100,7 @@ const contactMixin = <MixinBase extends CacheMixin & typeof PuppetSkeleton>(mixi
         try {
           // make sure the contact id has valid payload
           await this.contactPayload(query.id)
-          return [query.id]
+          return [ query.id ]
```

Comment:
Can you create another PR to fix the linting problem so that this PR can only contain the new logic for the Channel message to save our reviewing time?

Thanks!

Reply:
OK.

Reply:
https://github.com/wechaty/puppet/pull/218